pkg/cmd/render/render_test.go

```diff
@@ -173,6 +174,55 @@ data:
         projectID: openshift
         region: us-east1
     publish: External
+`
```

Comment:
(No action required) Maybe append the bootstrap-in-place entry to `clusterConfigMap`?